featuretools/primitives/utils.py

```diff
@@ -134,6 +204,65 @@ def _get_descriptions(primitives):
     return descriptions
 
 
+def _get_summary_primitives(primitives: List) -> Dict[str, int]:
+    """Provides metrics for a list of primitives."""
+    unique_input_types = set()
+    unique_output_types = set()
+    ct_multi_in = 0
+    ct_multi_out = 0
+    ct_extra_data = 0
+    ct_controllable = 0
+    logical_type_count = {
+        log_type: 0 for log_type in list(list_logical_types()["type_string"])
+    }
+    prim_input_ww_checks = {  # semantic tag for time index and logical types in inputs
+        "ct_semantic_time_idx": 0,
+        **logical_type_count,
+    }
+    for prim in primitives:
+        input_checks = set()
+        input_types = prim.flatten_nested_input_types(prim.input_types)
+        _check_input_types(input_types, input_checks, unique_input_types)
+        for check in list(input_checks):
+            prim_input_ww_checks[check] += 1
+
+        if len(prim.input_types) > 1:
+            ct_multi_in += 1
+
+        # checks if number_output_features is set as an instance variable
+        if "self.number_output_features =" in getsource(prim.__init__):
```

Comment:
While currently all multi-output primitives set `number_output_features` in the `__init__`, `number_output_features` could be set outside the `__init__` if the number of output features is constant.   We should probably check for that scenario.

Reply:
Got it! I'll add that in.